cmake/Modules/FindRTIConnextDDS.cmake

```diff
@@ -2314,4 +2393,42 @@ if(RTIConnextDDS_FOUND)
             RTIConnextDDS::cpp2_api
     )
 
+    # Cloud Discovery Service
+    create_connext_imported_target(
+        TARGET "cloud_discovery_service_c"
+        VAR "CLOUD_DISCOVERY_SERVICE_API_C"
+        DEPENDENCIES
+            RTIConnextDDS::routing_service_c
+    )
+
+    # Cloud Discovery Service CPP libraries are the C libraries + the CPP API
+    create_connext_imported_target(
+        TARGET "cloud_discovery_service_cpp2"
+        VAR "CLOUD_DISCOVERY_SERVICE_API_C"
+        DEPENDENCIES
+            RTIConnextDDS::cloud_discovery_service_c
+            RTIConnextDDS::cpp2_api
+    )
+
+    # Persistence Service C API
+    set(dependencies)
+
+    if(TARGET RTIConnextDDS::distributed_logger_c)
```

Comment:
Why this dependencies (distributed logger and rtisqlite) are conditional? If they are conditional they require different c_flags?

Reply:
It does not make any sense actually. Good point.